README.md

```diff
@@ -39,65 +39,4 @@ import devcycle_python_sdk
 
 ## Usage
 
-### Getting All Features
-```python
-    try:
-        # Get all features by key for user data
-        api_response = dvc.all_features(user)
-        print(api_response)
-    except ApiException as e:
-        print("Exception when calling DVCClient->all_features: %s\n" % e)
-    
-```
-
-### Get and Use Variable by Key
-To get values from your Variables, `all_variables()` is used to fetch variable values using the identifier `key` coupled with a default value. The default value can be of type string, boolean, number, or object.
-
-```python
-    key = 'key-test' # str | Variable key
-
-    try:
-        # Get variable by key for user data
-        variable = dvc.variable(user, key, 'default-value')
-        print("Variable value is: ", variable.value)
-    except ApiException as e:
-         print("Exception when calling DVCClient->variable: %s\n" % e)
-
-```
-
-### Getting All Variables
-To get values from your Variables, `all_variables()` is used to fetch variable values using the identifier `key` coupled with a default value. The default value can be of type string, boolean, number, or object.
-
-```python
-    key = 'key-test' # str | Variable key
-    
-    try:
-        # Get all variables for user data
-        variables = dvc.all_variables(user)
-        print(variables)
-    except ApiException as e:
-        print("Exception when calling DVCClient->all_variables: %s\n" % e)
-```
-
-### Track Event
-To POST custom event for a user
-```python
-
-    event = Event(
-        type="customEvent",
-        target="somevariable.key"
-    )
-    try:
-        # Post events to DevCycle for user
-        api_response = dvc.track(user, event)
-        print(api_response)
-    except ApiException as e:
-        print("Exception when calling DVCClient->track: %s\n" % e)
-```
-
-
-#### About this package
-
-This API client was generated by the [swagger-codegen](https://github.com/swagger-api/swagger-codegen) project.  By using the [swagger-spec](https://github.com/swagger-api/swagger-spec) from a remote server, you can easily generate an API client.
-
-- Package version: 1.0.0
\ No newline at end of file
+To find usage documentation, visit our [docs](https://docs.devcycle.com/docs/sdk/server-side-sdks/python).
```

Comment:
should point to: https://docs.devcycle.com/docs/sdk/server-side-sdks/python#usage